core/server/pom.xml

```diff
@@ -210,6 +210,12 @@
           <license>
             cHJvamVjdHxvcmcuYWxsdXhpby5hbGx1eGlvLWNvcmUtc2VydmVyfDIwMTgtMDQtMjh8dHJ1ZXwtMSNNQ3dDRkFFM0JGSnAwZ1FRNFFaNDd5SnlRd0x3ZGhmQUFoUU1ZMkJpWWVOamhOZndvQmU3UldMRVFzZnNGUT09
           </license>
+          <output>
+            <html>
+              <intro>${project.basedir}/intro.html</intro>
```

Comment:
should this file be in some resources/ folder? It seems a little strange to have it at the top level like this.